ha/util/health_generator/mock_health_event_publisher.py

```diff
@@ -0,0 +1,170 @@
+    cvg_count = Conf.get(_index, f'node>{node_id}>storage>num_cvg')
+    print([Conf.get(_index, f'node>{node_id}>storage>cvg[{cvg}]>name') for cvg in range(cvg_count)])
+
+def publish(args, conf_store: ConftStoreSearch = None) -> None:
+    """
+    publishes the message on the message bus.
+
+    Args:
+    conf_store: ConftStoreSearch object
+    node_id: machine_id value
+    config_file: config file path
+    """
+    print(f'inside publish, config file: {args.file}')
+
+FUNCTION_MAP = {
+                '-gdt' : get_data_nodes, '--get-data-nodes': get_data_nodes,
+                '-gs' : get_server_nodes, '--get-server-nodes': get_server_nodes
+                }
+
+def get_args():
+    """
+    Configures the command line arguments.
+    """
+    my_parser = argparse.ArgumentParser(prog='health_event_publisher',
+                                        usage='%(prog)s [options]',
+                                        description='Helps in publishing the mock health event',
+                                        epilog='Hope it is useful! :)')
+
+    subparsers = my_parser.add_subparsers()
+
+    my_parser.add_argument('-gdt', '--get-data-nodes', action='store_true', help='Get the list of data node ids')
+    my_parser.add_argument('-gs', '--get-server-nodes', action='store_true', help='Get the list of server node ids')
+
+    parser_publish = subparsers.add_parser('publish', help='Publish the message')
+    parser_publish.add_argument('-f', '--file', action='store', help='imput config file', required=True)
+    parser_publish.set_defaults(handler=publish)
+
+    parser_disks = subparsers.add_parser('get-disks', help='Displys the Disk Ids assosciated with the Node')
+    parser_disks.add_argument('-n', '--node-id', help='Node id for which disk id is required', required=True)
+    parser_disks.set_defaults(handler=get_disks)
+
+    parser_cvgs = subparsers.add_parser('get-cvgs', help='Displys the CVG Ids assosciated with the Node')
+    parser_cvgs.add_argument('-n', '--node-id', help='Node id for which disk id is required', required=True)
+    parser_cvgs.set_defaults(handler=get_cvgs)
+
+    args = my_parser.parse_args()
+    return args, my_parser
+
+
+if __name__ == '__main__':
+    if not is_container_env():
+        sys.exit('Please use this script in containerized environment')
+    option = None
+    args, parser_obj = get_args()
+    Conf.init()
+    file_to_load = f'{config_file_format}://{cluster_config_file}'
+    Conf.load(_index, file_to_load)
+    _conf_store = ConftStoreSearch(conf_store_req=False)
+
+    data_node_ids = get_data_nodes(conf_store=_conf_store)
+
+    if len(sys.argv) > 1:
+        option = sys.argv[1]
+    else:
+        parser_obj.print_help()
+        sys.exit(0)
+
+    if hasattr(args, 'handler'):
+        if hasattr(args, 'node_id') and args.node_id not in data_node_ids:
+            print('Please provide data node id to get disk and cvg id')
```

Comment:
[Suggestion] 
In case node id not present in data_node_ids, `Required data not found for given node id. Please check node ids list.` 
I can be better, @Madhura-08

Reply:
Right. Thanks. Done